src/WebSocketServer/WebSocketClient.cs

```diff
@@ -0,0 +1,86 @@
+using Neo.Json;
+using System;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Neo.Plugins
+{
+    internal class WebSocketClient : IDisposable, IEquatable<WebSocketClient>
+    {
+        public WebSocket Socket { get; init; }
```

Comment:
required and remove null checks?

Reply:
It is not null there. Hints the reason why no `?` and `init` on property. We don't want null.

Reply:
> It is not null there. Hints the reason why no `?` and `init` on property. We don't want null.

Yep, but it's used with ? inside the class

Reply:
Right and that reason, is for if client disconnect and the class becomes `disposed`.

Reply:
I marked the project as nullable and I detected it, I was going to fix it, I think that new projects should be nullable